docs/source/tutorial/production.md

```diff
@@ -15,16 +15,26 @@ In a production application, you should set up this publishing script as part of
 
 <h3 id="api-key">Get an Engine API key</h3>
 
-First, we need an Apollo Engine API key. Navigate to [Apollo Engine](https://engine.apollographql.com/), login, and click on New Service at the top. The prompt will instruct you to name your service. When you're finished, click Create Service. You'll see a key appear prefixed by `service:`. Copy that key so we can save it as an environment variable.
+First, we need an Apollo Engine API key. Navigate to [Apollo Engine](https://engine.apollographql.com/), login, and click on New Service at the top. The prompt will instruct you to name your service. Name the new service `space-explorer`. When you're finished, click Create Service. You'll see a key appear prefixed by `service:space-explorer:`. Copy that key so we can save it as an environment variable.
```

Comment:
We'll need to figure out another pattern here since the name must be globally unique.  That's to say, we can't recommend `space-explorer` here since once it's used (and it probably already is, by now) the next user will not be able to use it.